goltsmannn2/src/main.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+#include "hello_world.cpp"
+#include "../include/hello_world.hpp"
```

Comment:
Cmake takes care of this for you. This relative include is not needed and should not be used.